fdb-record-layer-core/src/main/java/com/apple/foundationdb/record/provider/foundationdb/OnlineIndexer.java

```diff
@@ -1850,39 +1909,42 @@ public static OnlineIndexer forRecordStoreAndIndex(@Nonnull FDBRecordStore recor
         /**
          * Only build if the index is disabled.
          */
-        BUILD_IF_DISABLED(false),
+        BUILD_IF_DISABLED,
         /**
-         * Build if the index is disabled; Continue build if the index is write-only.
+         * Build if the index is disabled; Continue build if the index is write-only - using the previous
+         * method that was used.
          * <p>
          * Recommended. This should be sufficient if current index data is not corrupted.
          * </p>
          */
-        BUILD_IF_DISABLED_CONTINUE_BUILD_IF_WRITE_ONLY(true),
+        BUILD_IF_DISABLED_CONTINUE_BUILD_IF_WRITE_ONLY,
         /**
          * Build if the index is disabled; Rebuild if the index is write-only.
          */
-        BUILD_IF_DISABLED_REBUILD_IF_WRITE_ONLY(false),
+        BUILD_IF_DISABLED_REBUILD_IF_WRITE_ONLY,
         /**
          * Rebuild the index anyway, no matter it it disabled or write-only or readable.
          */
-        FORCE_BUILD(false),
+        FORCE_BUILD,
         /**
-         * Error if the index is disabled, or continue to build if the index is write only. To use this option to build
-         * an index, one should mark the index as write-only and clear existing index entries before building. This
-         * option is provided to make {@link #buildIndexAsync()} (or its variations) behave same as what it did before
-         * version 2.8.90.0, which is not recommended. {@link #BUILD_IF_DISABLED_CONTINUE_BUILD_IF_WRITE_ONLY} should be adopted instead.
+         * Continue to build only if the index is write only and the requested method matches the previous one.
```

Comment:
Why removed "Error if the index is disabled"?

Reply:
I thought I was implied. Added back.